api/src/main/java/com/epam/pipeline/manager/cloud/CloudFacadeImpl.java

```diff
@@ -290,6 +300,31 @@ private AbstractCloudRegion getRegionByRunId(final Long runId) {
         }
     }
 
+    private void scaleUpNodeServices(final RunInstance instance) {
+        if (KubernetesConstants.WINDOWS.equals(instance.getNodePlatform())) {
```

Comment:
I'd suggest to move this logic to kube-related classes

Reply:
Done